src/otx/core/engine/test.py

```diff
@@ -33,9 +33,6 @@ def test(cfg: TrainConfig) -> tuple[Trainer, dict[str, Any]]:
     trainer: Trainer = hydra.utils.instantiate(cfg.trainer)
 
     log.info("Starting testing!")
-    if cfg.checkpoint is None:
```

Comment:
So, in this case, users can `test` without checkpoint?

Reply:
Yes, just with pretrained weights. It is useful sometimes to check model and what the random score or just to test pre-trained weights